### PR TITLE
Nc games 3

### PR DESCRIPTION
**Pull request by FelicityRC**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,17 @@
 # Northcoders House of Games API
 
+<<<<<<< HEAD
```
Comment (alexdevdavis) [inline README.md:3]: the 
<<<<< HEAD 
and  
>>>>>>> main
bits are still showing in this file, by the looks of things ... I'd suggest having a root through and clearing out some repeated/unwanted bits

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+describe.only("GET: /api/categories", () => {
+  test("200: responds with array of category objects with slug & description properties", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then((response) => {
+        const {
+          body: { body },
+        } = response;
+        body.forEach((property) => {
```
Comment (alexdevdavis) [inline __tests__/app.test.js:28]: when using forEach on an array, ensure to check the length of the array so you're not getting a false positive

**Review by alexdevdavis — APPROVED** [review]
Nicely done, Felicity! PR approved. 
See comments to inform a refactor (perhaps for a quieter moment)

File: __tests__/app.test.js (hunk 2 above)
Comment (FelicityRC) [reply __tests__/app.test.js]: Thank you - done!

File: controllers/controllers.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const { fetchCategories } = require("../models/models")
+
+function getCategories(request, response, next) {
+    request.query;
+    fetchCategories()
+    .then((games) => {
+        response.status(200).send({ body: games })
```
[the remark this answers is not in the document]
Comment (FelicityRC) [reply controllers/controllers.js]: Thank you - done!

**Pull request merged**